### PR TITLE
[bitnami/discourse] Update strategy added

### PR DESCRIPTION
**Pull request by utkudarilmaz**
**Description of the change**

Deployment update strategy added.

**Benefits**

With flexible update strategy users can change their deployment strategies about own benefits.

**Possible drawbacks**

There is no drawbacks.

**Applicable issues**

**Additional information**

When users using RWO PV's and default strategy (RollingUpdate), the deploy process is stucking because the old pod never release PVC before new Pod becaming healthy. So if users can change update strategy to **Recreate** stucking issue will fixed.

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

### CODE REVIEW

File: bitnami/discourse/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: discourse
 description: A Helm chart for deploying Discourse to Kubernetes
-version: 0.4.1
+version: 0.5.1
```
Comment (dani8art) [inline bitnami/discourse/Chart.yaml:4]: It would be:

```suggestion
version: 0.5.0
```

File: bitnami/discourse/README.md [hunk 2]
```diff
@@ -81,6 +81,7 @@ The following table lists the configurable parameters of the Discourse chart and
 | `persistence.existingClaim`          | Name of an existing PVC to reuse                                                           | `nil`                                                        |
 | `persistence.accessMode`             | PVC Access Mode (RWO, ROX, RWX)                                                            | `ReadWriteOnce`                                              |
 | `persistence.size`                   | Size of the PVC to request                                                                 | `10Gi`                                                       |
+| `updateStrategy`                     | Update strategy of deployment                                                              | `{}`                                                         |
```
Comment (dani8art) [inline bitnami/discourse/README.md:84]: We should add a default value for `updateStrategy`, maybe the simplest one.

**Review by dani8art — CHANGES_REQUESTED** [review]
Hi @utkudarilmaz thank you so much for the PR. Please, take a look at my comments.

**Review by dani8art — APPROVED** [review]
LGTM

**Comment by utkudarilmaz** [comment]
@dani8art thanks, best regards!

**Pull request merged**